File: Hibernate & JPA/Code/02/Supplier.java
package org.example;

import javax.persistence.*;

@Entity
public class Supplier {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private int supplierID;
    private String companyName;
    private String street;
    private String city;

    public Supplier() {}

    public Supplier(String companyName, String street, String city) {
        this.companyName = companyName;
        this.street = street;
        this.city = city;
    }
}

File: Entity Framework/Kod/02/Supplier.cs
namespace ŁukaszDydekEFProducts
{
    internal class Supplier
    {
        public int SupplierID { get; set; }
        public string CompanyName { get; set; }
        public string Street { get; set; }
        public string City { get; set; }

        public Supplier(string CompanyName, string Street, string City) 
        { 
            this.CompanyName = CompanyName;
            this.Street = Street;
            this.City = City;
        }
    }
}
